yarn.lock

```diff
@@ -1525,7 +1525,7 @@ ecc-jsbn@~0.1.1:
 
 "eclipse-che@git://github.com/eclipse/che#master":
   version "0.0.0"
-  resolved "git://github.com/eclipse/che#417eb0347356124eb636e979e8bed48b491694e8"
+  resolved "git://github.com/eclipse/che#8c9740f6f593977cfc514629bcb2e06287536d6e"
```

Comment:
Does this change really needed?

Reply:
was changed automatically, IDK the reason.